README.md

```diff
@@ -1,88 +1,79 @@
 # PromoTube
 
 PromoTube is a webapp that allows users to easily find promo codes and affiliate links that go with their favorite creators or products.
+It uses the YouTube Data API to find videos from a given channel. We then parse the video descriptions and return the found promo codes.
```

Comment:
PromoTube uses the YouTube Data API to scrape promo-codes from descriptions given channel's URL or business's name.  What do you think?